sporks.json

```
@@ -1309,6 +1309,45 @@
       }
     },
     "testnet": {
+      "testnet48": {
+        "id": 48,
+        "live": true,
+        "name": "Testnet48",
+        "sporkTime": "2023-10-25T14:30:00Z",
+        "rootHeight": "127720465",
+        "rootParentId": "2b30e75bd857f898456dcb296bea4b8bc8001cab7062eeee9e47876411b36d76",
+        "rootStateCommitment": "f2e5ebfca2fd519e49f7bd85bea81e92eeaa85a705b3376d8534e9c9649da710",
+        "gitCommitHash": "692969b1718b3d21f95ee7f66e5061623d99e599",
+        "stateArtefacts": {
+          "gcp": {
+            "rootCheckpointFile": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48-execution/public-root-information/root.checkpoint",
+            "rootProtocolStateSnapshot": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json",
+            "rootProtocolStateSnapshotSignature": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json.asc",
+            "nodeInfo": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/node-infos.pub.json"
+          },
+          "s3": {
+            "rootCheckpointFile": "",
+            "rootProtocolStateSnapshot": "",
+            "nodeInfo": "",
+            "executionStateBucket": ""
+          }
+        },
+        "tags": {
+          "flow-go-tag": "v0.32.3",
+          "flow-dps-tag": "v0.32.3",
+          "cadence-tag": "v0.42.1",
+          "docker-tag": "v0.32.3"
+        },
+        "accessNodes": [
+          "access-001.devnet48.nodes.onflow.org:9000"
+        ],
+        "seedNodes": [
+          {
+            "address": "access-001.devnet48.nodes.onflow.org:3570",
```

this should be the access-003 node. we just transition it to 001 for past sporks.
```suggestion
            "address": "access-003.devnet48.nodes.onflow.org:3570",
```


this is confusing and I'll look to just switch it all over to use AN1 in the future